osscameroon/jobsika
# Order rating by createdat

This pull request order ratings by createdat DESC when get ratings. Related to #306 

Steps to verify:

   - move to the backend folder with cd ./backend
   -  run make start-postrges to run an initialized database
   - run make run to launch the api
   - then run the command

1️⃣Add a rating for Njombe

```shell
curl -s -X POST 'http://localhost:7000/ratings' -H 'Content-Type: application/json' -d '{"city": "Njombe","comment": "my comment","company_name": "Njombe","job_title": "Dev","rating": 2,"salary": 1200,"seniority": "CV"}'
```

you should see something like this 👇
```json
{"message":"created"}
```

2️⃣Get list of ratings 
```shell
curl -s -X GET "http://localhost:7000/ratings?page=1&limit=500" | jq | head -n 16
```
you should see the ratings that we just add

```json
{                                        
  "hits": [                              
    {                                    
      "salary_id": 121,                  
      "company_id": 0,                   
      "company_rating_id": 49,           
      "rating": 0,                       
      "salary": 1200,                    
      "company_name": "",                
      "seniority": "CV",                 
      "comment": "",                     
      "job_title": "Dev",                
      "country": "Cameroon",             
      "city": "Njombe",                  
      "createdat": "2022-05-16T00:00:00Z"
    },
```
